specification/paths/MultiColliShipments.json

```diff
@@ -0,0 +1,281 @@
+                          "$ref": "#/components/schemas/Price"
+                        }
+                      }
+                    }
+                  },
+                  "label": {
+                    "type": "object",
+                    "properties": {
+                      "mime_type": {
+                        "type": "string",
+                        "example": "application/pdf",
+                        "default": "application/pdf",
+                        "description": "Requested `mime_type` for the label of this shipment. Available mime_types can be found on the carrier resource related to the chosen service/contract."
+                      },
+                      "size": {
+                        "type": "string",
+                        "example": "A6",
+                        "default": "A6",
+                        "description": "Requested size for the label of this shipment. Available label sizes can be found on the carrier resource related to the chosen service/contract."
+                      }
+                    }
+                  },
+                  "service_code": {
+                    "type": "string",
+                    "example": "carrier-parcel-next-day",
+                    "description": "If an applicable service is found, it will be set as relationship. If no contract relationship is defined, the cheapest contract for this service will be applied."
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "responses": {
+      "201": {
+        "description": "The shipment is created and registered.",
+        "headers": {
+          "Idempotency-Replay": {
+            "description": "Present with value `true` when the response is replayed based on the provided `Idempotency-Key` header.",
+            "schema": {
+              "type": "string"
+            }
+          }
+        },
+        "content": {
+          "application/vnd.api+json": {
+            "schema": {
+              "type": "object",
+              "required": [
+                "data"
+              ],
+              "properties": {
+                "data": {
+                  "allOf": [
+                    {
+                      "$ref": "#/components/schemas/Shipment"
+                    },
+                    {
+                      "required": [
+                        "id",
+                        "links"
+                      ],
+                      "properties": {
+                        "relationships": {
+                          "required": [
+                            "contract",
+                            "files",
+                            "service",
+                            "shipment_status",
+                            "shop"
+                          ]
+                        }
+                      }
+                    }
+                  ]
+                },
+                "meta": {
```

Comment:
This might not be needed since the contents of the labels is in the colli relationship